zip-0228.rst

```diff
@@ -2,12 +2,530 @@
++-----------------------------+--------------------------+--------------------------------------------------+---------------------------------------------------------------------+
+|``32``                       |``anchorOrchard``         |``byte[32]``                                      |As defined in Section 7.1 of the Protocol                            |  
+|                             |                          |                                                  |Specification [#protocol-txnencoding]_.                              |  
++-----------------------------+--------------------------+--------------------------------------------------+---------------------------------------------------------------------+
+|``varies``                   |``sizeProofsOrchard``     |``compactSize``                                   |As defined in Section 7.1 of the Protocol                            |  
+|                             |                          |                                                  |Specification [#protocol-txnencoding]_.                              |  
++-----------------------------+--------------------------+--------------------------------------------------+---------------------------------------------------------------------+
+|``sizeProofsOrchard``        |``proofsOrchard``         |``byte[sizeProofsOrchard]``                       |The aggregated zk-SNARK proof for all Actions in the Action Group.   |  
++-----------------------------+--------------------------+--------------------------------------------------+---------------------------------------------------------------------+
+|``4``                        |``timeLimit``             |``uint32``                                        |The block number (in the future) after which the Actions of the      |  
+|                             |                          |                                                  |Action Group become invalid and should be rejected by consensus.     |  
++-----------------------------+--------------------------+--------------------------------------------------+---------------------------------------------------------------------+
+
+.. figure:: zip-0228-images/zip-0228-swap-bundle.png
+    :width: 1000px
+    :align: center
+    :figclass: align-center
+
+    An example Zcash transaction with a Swap Bundle (simplified, for reference only).
+
+Spend Authorization Signature Changes
+-------------------------------------
+
+The spend authorization signature is computed in the manner specified in Section 4.15 of the protocol specification [#protocol-spendauthsig]_, except that the signature is generated over the Action Group Hash, :math:`\mathsf{AGHash}`, which is defined to be
+:math:`\mathsf{AGHash} := \mathsf{BLAKE2b}\texttt{-}\mathsf{256}(\texttt{"ZcashActionGroup"}, \mathsf{preAuthActionGroup})`.
+
+This change ensures that no changes are required in the Action Statement to be proved.
```

Comment:
I see, I was going for the term in section 4.17.4 of the protocol specification, but I see that the specification PDF also does not capitalise Statement when using it in a sentence. I'll add a reference to the section so that it is clear what we mean by the term though.